components/01-globals/label/Label.vue

```diff
@@ -1,11 +1,11 @@
 <template>
   <component
-    :is="tag ? tag : 'label'"
-    :class="{
-      label: tag,
+    :is="tag"
+    :class="[customClass, {
```

Comment:
We don't need to add a new prop to handle adding a class if it's related to the root level element, b/c it will take the normal `class` value here.